### ISSUE TITLE
extend configuration with "no_proxy"

### ISSUE THREAD

**Issue by m777** [bower/bower#857]
We need an extension of the configuration with the property "no_proxy". We have to be able to go to external bower repositories and an internal one. It would be nice to have this advanced proxy configuration so that remote and local repositories can be used easily in parallel.

**Comment by jfbibeau** [comment]
+1 this is much needed. Any news?

**Comment by Odi55555** [comment]
We're using cntlm at the moment, but we would appreciate if bower would respect the env variables like @abovethewater suggested it. I just wanted to note that it works fine with npm and env variables. Maybe we could use the same solution?

**Comment by sheerun** [comment]
This can be now implemented as custom [Pluggable Resolver](http://bower.io/docs/pluggable-resolvers/) that overwrites Bower's native one.

Bower's core time doesn't have time to implement it, but I'd really like to see it!

**Comment by pdxjohnny** [comment]
@sheerun If I implement this will you include it in the core? Because it would be nice to have as standard

**Comment by sheerun** [comment]
@pdxjohnny Selection of Pluggable Resolvers will be included in Bower 2.0
